docs/learning/tutorials/develop-web-viewer.md

```diff
@@ -1,19 +1,22 @@
 ## Developing a web viewer
 
-### Setup
-- [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone imodeljs-samples repo](https://github.com/imodeljs/imodeljs-samples)
-- Have access to a cloud hosted iModel. If you do not have access to one, follow one of our tutorials to [create an online iModel]($docs/learning/tutorials/index.md)
+### Develop
+- [Install necessary prerequisites]($docs/getting-started/development-prerequisites).
+- From a terminal, `npx create-react-app your-app-name --template @bentley/itwin-viewer --scripts-version @bentley/react-scripts`.
+    - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
+- Open the `your-app-name` directory in VS Code.
+- Add a valid contextId and iModelId for your user to the REACT_APP_TEST_CONTEXT_ID and REACT_APP_TEST_IMODEL_ID variables in the .env file within the application's root directory.
+  - This will be used for initial development. The idea is that it would be replaced by a proper model selection process in a production application.
+- From a terminal at your application's root directory, `npm start`. This will serve the application with live reloading.
+- Add/Update/Remove files as needed for your use case. If running `npm start` while making changes, your application should re-compile and reload.
+- The viewer can be modified via the Viewer component in the App.tsx file. Visit the [iTwin Viewer React](https://www.npmjs.com/package/@bentley/itwin-viewer-react) documentation for more information.
+- Visit the [README.md](https://github.com/imodeljs/itwin-viewer/blob/master/packages/modules/cra-template-itwin-viewer/template/README.md) file within the root directory of your application for additional development information.
 
 ### Build
-- Open the cloned repo in VS Code
-- `cd interactive-app`
-- `cd basic-viewport-app`
-- Edit interactive-app\basic-viewport-app\\.env.local
-    > `imjs_test_imodel` = The name of your iModel<br/>
+- From a terminal at your application's root directory, `npm run build`. This will create a deployment-ready build in the "build" folder within the application's root directory. It is not necessary to build the application during development.
 
-- `npm install`
-- `npm run build`
-
-### Run
-- `npm run start:servers`
+### Useful Links

```

Comment:
Lets also have a "Next Steps" section at the end of this that points at the Sample Showcase.

Reply:
Added